qa/tasks/cephfs/test_mirroring.py

```diff
@@ -1305,3 +1305,13 @@ def test_local_and_remote_dir_root_mode(self):
         self.disable_mirroring(self.primary_fs_name, self.primary_fs_id)
         self.mount_a.run_shell(["rmdir", "l1/.snap/snap0"])
         self.mount_a.run_shell(["rmdir", "l1"])
+
+    def test_get_set_mirror_attributes(self):
```

Comment:
Also include test for the ceph.mirror.info

Reply:
@vshankar I think it's already there [1] ?
[1] https://github.com/ceph/ceph/blob/main/src/test/pybind/test_cephfs.py#L161